cli/run/commandPlugins.ts

```diff
@@ -8,11 +8,14 @@ export async function addCommandPluginsToInputOptions(
 	inputOptions: InputOptions,
 	command: Record<string, unknown>
 ): Promise<void> {
+	if (!Array.isArray(inputOptions.plugins)) {
```

Comment:
Actually at this point, it is guaranteed that `plugins` is an array, I just forgot to reflect it in the types. The reason is that `mergeInputOptions` runs first on the options that are passed here, and that one ensures `plugins` is an array. A nicer solution would be the following:

In `types.d.ts`, introduce a new type like

```ts
export interface InputOptionsWithPlugins extends InputOptions {
	plugins: (Plugin | null | false | undefined)[];
}
```

and change

```ts
export interface MergedRollupOptions extends InputOptionsWithPlugins {
	output: OutputOptions[];
}
```

Then use the new type in `mergeInputOptions` and change the type in `addCommandPluginsToInputOptions` from `InputOptions` to the more correct `MergedRollupOptions`.